src/mgr/MgrClient.cc

```diff
@@ -331,6 +332,11 @@ void MgrClient::_send_report()
         const PerfCounters::perf_counter_data_any_d &ctr,
         const PerfCounters &perf_counters)
     {
+      // FIXME: We don't send labeled perf counters on the wire to the mgr currently.
+      auto label_range = ceph::perf_counters::key_labels(perf_counters.get_name());
```

Comment:
These lines are too long:
```suggestion
      // FIXME: We don't send labeled perf counters to the mgr currently.
      auto label_range =
          ceph::perf_counters::key_labels(perf_counters.get_name());

```